### PR TITLE
chore: create 'tombstone Collection' utility

### PR DESCRIPTION
**Pull request by danieljhegeman**
## Reason for Change

- #5186 

## Changes

- add logic to the `tombstone_collection` function skeleton in `tombstones.py`
- force `dev`, `staging`, or `prod` value for `--deployment` in general usage of the cxg_admin script.
- use Secrets to populate happy env variable(s) necessary for Business / Persistence logic

## Testing steps

- tested manually, adhoc, tombstoning Collections in dev and staging. Confirmed all entities tombstoned and public assets deleted from public datasets bucket.

## Notes for Reviewer

- try it out! In dev 😅

### CODE REVIEW

File: scripts/populate_db.py [hunk 1]
```diff
@@ -22,10 +27,11 @@
 def run_db_stuff(create_schema, recreate_db, populate_data, drop_db):
     # Create schema.
     if create_schema:
+        print(CorporaDbConfig().database_uri)
         engine = create_engine(CorporaDbConfig().database_uri)
         if not database_exists(engine.url):
             print("Database does not exist, creating database")
-            create_database(engine.url)
+            # create_database(engine.url)
```
Comment (nayib-jose-gloria) [inline scripts/populate_db.py:34]: did you mean to comment this out?

File: scripts/populate_db.py [hunk 2]
```diff
@@ -35,7 +41,7 @@ def run_db_stuff(create_schema, recreate_db, populate_data, drop_db):
         engine = create_engine(CorporaDbConfig().database_uri)
         if database_exists(engine.url):
             print("Database exists, dropping database")
-            drop_database(engine.url)
+            # drop_database(engine.url)
```
Comment (nayib-jose-gloria) [inline scripts/populate_db.py:44]: did you mean to comment this out?

Comment (danieljhegeman) [reply scripts/populate_db.py]: @nayib-jose-gloria thanks for catching, I had meant to clean this up after some adhoc testing. It isn't used, per se, but I've reverted it to the previous state. 👍

File: scripts/cxg_admin.py [hunk 3]
```diff
@@ -44,9 +50,20 @@ def cli(ctx, deployment):
     You must first set DEPLOYMENT_STAGE as an env var before running
 
     """
+    if deployment == "test":
```
[the remark this answers is not in the document]
Comment (danieljhegeman) [reply scripts/cxg_admin.py:53]: The unit tests I've added for the scripts require that the rest of the code in this function not be run. Separate configurations.

**Review by nayib-jose-gloria — APPROVED** [review]

**Pull request merged**